tests/test_methods.py

```diff
@@ -83,9 +83,9 @@ def test_reward_func():
     env.reset()
     # self.state = self.self.init_state()
 
-    # test no trap laid for one timestep when no crab
+    # test no trap laid for one timestep when no crab (still potentially negative due to environmental deterioration)
     action = np.array([-1, -1, -1])
-    assert env.reward_func(action) >= 0 # are we expecting non-negative when no traps laid when no crabs
+    assert env.reward_func(action) <= 0
```

Comment:
haha good catch, thanks

Reply:
haha yep! It looks like the warning comes from SB3's env checking on the 'time series env', which collects a time-series of env observations in a 2-D array. It suggests we should flatten it, but I will leave it for the next PR since we might anyways switch to using SB3's frame_stack for this.